.github/workflows/ci.yaml

```diff
@@ -28,26 +28,6 @@ jobs:
       - name: Run linters
         run: tox run -e lint
 
-  lib-check:               
```

Comment:
Do we not need this anymore?

Reply:
Library changes will be pulled from upstream